pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -105,6 +112,7 @@ def add_lazy_oa_cuts(self, target_model, dual_values, solve_data, config, opt,
             # Equality constraint (makes the problem nonconvex)
             if constr.has_ub() and constr.has_lb() and constr.upper == constr.lower:
                 sign_adjust = -1 if solve_data.objective_sense == minimize else 1
+                # !!THIS SEEMS LIKE A BUG!! - mrmundt #
```

Comment:
@jsiirola  - `rhs` has never been assigned anywhere, so the statement `rhs = foo if bar else rhs` doesn't work if the `else` is triggered.

Reply:
@bernalde: thoughts?

Reply:
Same case here, unreachable bug, but a bug. Should be `rhs = constr.lower` @ZedongPeng